Enhance Live Tail documentation with keyword usage note (#6279)
SumoLogic/sumologic-documentation: docs/search/live-tail/filter-live-tail.md

The **Run** button changes to **Running**, the new query runs, and the screen automatically scrolls with the results.

:::note

> Ensure that you use appropriate keywords for effective Live Tail usage and avoid exceeding the limitations described in the [About Live Trail](/docs/search/live-tail/about-live-tail/#limitations).

Reviewer: ```suggestion
Ensure that you use appropriate keywords for effective Live Tail usage, and ensure your query does not exceed the limitations described in [About Live Tail](/docs/search/live-tail/about-live-tail/#limitations).
```
